client/txn_test.go

```diff
@@ -75,7 +75,7 @@ func TestTxnRequestTxnTimestamp(t *testing.T) {
 		{makeTS(20, 1), makeTS(20, 1)},
 	}
 
-	testIdx := 0
+	var testIdx int
```

Comment:
It probably prefers the implicit zero value over the explicit. The price to pay for having a lot of automated code checkers...